src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -106,15 +134,56 @@ public void setIgnoreOverridden(boolean ignoreOverridden) {
     @Override
     protected boolean mustCheckName(DetailAST ast) {
         boolean checkName = true;
+        final boolean isDefault = scope == Scope.ANONINNER && excludeScope == null;
```

Comment:
This reference to ANONINNER is necessary to implement your suggestion about not doing anything if the user has not overridden the defaults (the comment disappeared for some reason).